src/pages/installation.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: ~/layouts/Main.astro
+---
+
+# Installing Astro
+
+Welcome to Astro, we are really excited that you are here and would like to assist you in getting up and started with Astro in no time.
+
+<!--
+    Quick Install
+    Manual Install
+    Troubleshooting
+ -->
+
+## Preflight
+
+Before we begin, let us check to make sure that everything is in order.
+
+To use Astro you need to have Node.js already installed and working on your machine. If you haven't got Node.js installed, you can download it from their site.
+
+⚠️ **Important**: Astro is built and uses ESM modules as default. This is only supported from Node versions 12 and above,. However, to use Astro you would need to have at least version `14.16.1` installed on your machine.
+
+You would also need a package manager to install and use Astro.
+
+Node.js ships their own package manager during installation, so you would have **npm** on your system. You are not restricted to using only npm, you can use your preferred package manager if you so wish:
+    - [npm](https://docs.npmjs.com/about-npm)
+    - [Yarn](https://yarnpkg.com/getting-started)
+    - [pnpm](https://pnpm.io/installation)
+
+We ask that your table and chairs and in upright postilions, and you remain seated until after we take-off.
```

Comment:
> We ask that your table and chairs and in upright postilions
Something feels amiss, maybe :
- We ask that **you put** your table and **chairs in** upright postilions
- We ask that your table and chairs **are** in upright postilions